server/lib/util.py

```diff
@@ -345,6 +345,14 @@ def get_nl_chart_titles():
   return chart_titles
 
 
+# Returns display titles for properties used in NL
+def get_nl_prop_titles():
```

Comment:
comment on returned structure, and use ret-type?

Reply:
done